### PR TITLE
Social sharing

### PR DESCRIPTION
**Pull request by davidtrussler**
Dough component for Social Sharing icons to be re-used across main site and tools etc.

### CODE REVIEW

File: assets/stylesheets/components/optional/_social_sharing_icons.scss [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+.social-sharing__icon {
+  height: 33px;
+  width: 55px;
+}
+
+.social-sharing__item--facebook {
+  background: $color-facebook;
+
+  .social-sharing__item__icon:hover {
```
Comment (benbarnett) [inline assets/stylesheets/components/optional/_social_sharing_icons.scss]: You can use a SASS helper here to just make this line:

```css
&:hover {
```

File: assets/stylesheets/components/optional/_social_sharing_icons.scss [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    background: $color-facebook-hover;
+  }
+}
+
+.social-sharing__item--twitter {
+  background: $color-twitter;
+
+  .social-sharing__item__icon:hover {
```
Comment (benbarnett) [inline assets/stylesheets/components/optional/_social_sharing_icons.scss]: Same here re helper

File: assets/stylesheets/components/optional/_social_sharing_icons.scss [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    background: $color-twitter-hover;
+  }
+}
+
+.social-sharing__item--email {
+  background: $color-email;
+
+  .social-sharing__item__icon:hover {
```
Comment (benbarnett) [inline assets/stylesheets/components/optional/_social_sharing_icons.scss]: & here

**Comment by benbarnett** [comment]
Nice :+1:

File: docs/helpers/SocialSharingIcons.erb [hunk 4]
```diff
@@ -0,0 +1 @@
+<%= social_sharing_icons title: 'Share this article', text: 'Text to appear in message', url: 'http://www.madewithdough.org.uk/', label_facebook: 'Share this article on Facebook', label_twitter: 'Share this article on Twitter', label_email: 'Share this article by Email' %>
```
Comment (tomas-stefano) [inline docs/helpers/SocialSharingIcons.erb]: Can we rename this file to use underscore? Can you rename to social_sharing_icons.html.erb, please?

Comment (benbarnett) [inline docs/helpers/SocialSharingIcons.erb]: @tomas-stefano It needs to stay like this as it is rendered automatically – the filename is matched to the corresponding file in `docs/`

Comment (tomas-stefano) [inline docs/helpers/SocialSharingIcons.erb]: Ohh this is the docs. Nevermind.

Comment (benbarnett) [inline docs/helpers/SocialSharingIcons.erb]: P.S. This is how the [madewithdough](http://www.madewithdough.org.uk/) site renders its documentation

**Comment by tomas-stefano** [comment]
:+1:

File: docs/helpers/SocialSharingIcons.erb (hunk 4 above)
Comment (tomas-stefano) [inline docs/helpers/SocialSharingIcons.erb]: Thanks @benbarnett My bad!

**Pull request merged**